src/material-experimental/mdc-slider/slider.spec.ts

```diff
@@ -650,6 +794,22 @@ function slideToValue(slider: MatSlider, value: number, thumbPosition: Thumb, is
   dispatchPointerOrTouchEvent(sliderElement, PointerEventType.POINTER_UP, endX, endY, isIOS);
 }
 
+/**
+ * Mimics changing the slider value using arrow keys.
+ *
+ * Dispatching keydown events on inputs do not trigger value changes. Thus, to mimic this behavior,
+ * we manually change the slider inputs value and then dispatch a change event (which is what the
+ * MDC Foundation is listening for & how it handles these updates).
+ */
+function changeValueUsingArrowKeys(slider: MatSlider, arrow: number, thumbPosition: Thumb) {
+  const input = slider._getInput(thumbPosition);
+  const value = arrow === RIGHT_ARROW
+    ? input.value + slider.step
+    : input.value - slider.step;
+  input._hostElement.value = value.toString();
+  dispatchEvent(input._hostElement, new Event('change'));
```

Comment:
Not mandatory, but you should be able to simplify this further using `dispatchFakeEvent('change')`.

Reply:
Whoops, I missed this comment. Since this is such a tiny change & this is all on a feature branch, I'm just going to commit a fixup with this change directly on the `mdc-slider` branch and then squash it once it passes CircleCI checks